### DISCUSSION TITLE
Examples Developer Experience

### DISCUSSION THREAD

**Discussion by epreston** [playcanvas/engine#5636] [General]

**Comment by kungfooman** [comment]
For everyone who hasn't seen Bret Victor yet, give it 10 minutes for inspiration:

https://www.youtube.com/watch?v=PUv66718DII

I gained a bit of experience in the past creating a live coding session like that, and I'm working towards additions to this experience you have probably not seen anywhere else before, but that will first be possible after #5555 is done :sweat_smile:
